app/components/wallet/PaperWalletImportDialog.scss

```diff
@@ -42,6 +42,26 @@
       width: 275px;
     }
 
+    :global {
+      .SimpleFormField_root {
+        &.SimpleInput_errored {
+          margin-bottom: 21px;
+
+          .SimpleFormField_error {
```

Comment:
Maybe you could move that duplicated styles to a sass mixin called `placeFormFieldErrorBelowInput`